apps/showcase/pom.xml

```diff
@@ -145,6 +146,48 @@
 
     <build>
         <plugins>
+            <plugin>
+              <groupId>org.apache.maven.plugins</groupId>
+              <artifactId>maven-dependency-plugin</artifactId>
+              <executions>
+                <execution>
+                  <?m2e ignore?>
+                  <id>copy-dependencies</id>
+                  <phase>package</phase>
+                  <goals>
+                    <goal>copy</goal>
```

Comment:
I'm probably missing something here but why do we need to explicitly copy these jars?

Reply:
@kusalk  I struggled when doing this to figure out how the jetty-maven-plugin was building its classpath.  It was not including the 4 jars we generate from the Eclipse Transformer, so the showcase integration tests all would fail.  The solution I found was to use the Jetty "extraclasspath" feature (see same pom.xml lines 236ff) in conjunction with copying the jars as you see here.  In my ignorance I may have passed up a more-elegant solution!

Reply:
It should be possible to play with `<exclusion>` and explicitly defining the re-packaged jakarta libs, but this can be done after merge